### PR TITLE
ReactART:-Added unit Tests in the ReactART, increasing the code coverage

### PR DESCRIPTION
**Pull request by Biki-das**
This PR added unit tests around the `ReactART` which has also increased the coverage

Test and coverage before the PR:-
Windows PowerShell 27-01-2022 11 23 47 AM
Windows PowerShell 27-01-2022 11 26 21 AM

Test and coverage after the PR:-

Windows PowerShell 27-01-2022 11 21 16 AM
Windows PowerShell 27-01-2022 11 26 08 AM

### CODE REVIEW

**Comment by Biki-das** [comment]
@lunaruan could you have a look on this?

**Comment by Biki-das** [comment]
this should be good to go, i guess this has a good amount of test coverage , cc @lunaruan

**Comment by Biki-das** [comment]
@rickhanlonii , ping

File: packages/react-art/src/__tests__/ReactART-test.js [hunk 1]
```diff
@@ -440,6 +440,60 @@ describe('ReactARTComponents', () => {
     expect(rectangle.toJSON()).toMatchSnapshot();
   });
 
+  it('should generate a <Shape> with positive width when width prop is negative', () => {
```
Comment (Biki-das) [inline packages/react-art/src/__tests__/ReactART-test.js:443]: this adds a lot of tests

**Comment by Biki-das** [comment]
@gaearon could you give a feedback

**Comment by Biki-das** [comment]
@sebmarkbage could you review?

**Comment by sebmarkbage** [comment]
Sure.

**Pull request merged**